ods_ci/utils/scripts/rosa/rosa.py

```diff
@@ -21,6 +23,21 @@ def __init__(self, args={}):
         self.rosa_version = args.get("rosa_version")
         self.channel_name = args.get("channel_name")
 
+    def set_rosa_version(self):
+        version_match = re.match(r"(\d+\.\d+)\-latest", self.rosa_version)
+        if version_match is None:
+            log.info(f"Using the rosa version given by user: {self.rosa_version}")
+            return
+        log.info(f"User provided {self.rosa_version}, trying to determine the appropriate latest version for ROSA")
+        version = version_match.group(1)
+        latest_version_cmd = (
+            f"rosa list versions --channel-group {self.channel_name} | "
+            f"awk '{{print $1}}' | grep -w '^{re.escape(version)}*' | head -n1"
+        )
```

Comment:
I'd run `rosa list versions --channel-group {self.channel_name}` as subprocess and then I'd process the output in Python, not with awk, grep, and head

Reply:
In case I need the command output for more usages I can agree  
but in this case, I like it in "one line" of code.
Thanks @jiridanek